.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CheckStyle-IDEA">
+    <option name="configuration">
+      <map>
+        <entry key="active-configuration" value="PROJECT_RELATIVE:$PROJECT_DIR$/.mvn/checkstyle.xml:.mvn/checkstyle.xml" />
+        <entry key="checkstyle-version" value="8.22" />
+        <entry key="copy-libs" value="false" />
+        <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
```

Comment:
is it an order preference?

Reply:
No, see the key `active-configuration`. Those 2 locations (0,1) are bundled and cannot be removed.